src/Spiffy.Monitoring/Config/InitializationApi.cs

```diff
@@ -39,6 +41,17 @@ public InitializationApi()
         /// values, <code>false</code> otherwise.
         /// </returns>
         public bool RemoveNewlines { get; set; }
+        
+        /// <summary>
+        /// Values over this length will be deprioritized in the <see cref="LogEvent.Message"/>.
+        /// Defaults to 1024.
+        /// </summary>
+        /// <remarks>
+        /// In some logging scenarios, long values can result in some key/value pairs being cut off.
+        /// Key/value pairs with values whose length exceeds this value will be output after those
+        /// pairs whose values do not.
+        /// </remarks>
+        public int DeprioritizedValueLength { get; set; }
```

Comment:
nitpick: you could initialize here to keep the value closer to the documentation

Reply:
Done in new commit